src/detail-page/components/TabJobContent/JobTab.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { Button, CircularProgress, Input } from '@material-ui/core';
+import JobListItem from './JobListItem';
+import useFetchJobs from './useFetchJobs';
+import Job from '../../../data/Jobs';
+import hiringPicture from '../../../icons/hiring.svg';
+
+const SearchInput = styled(Input)`
+  width: 250px;
+  height: 35px;
+  border: 1px #CFCFCF solid;
+  margin: ${props => props.theme.spacing(5)}px 0 ${props => props.theme.spacing(1.5)}px ${props => props.theme.spacing(1.5)}px ;
+  padding-left: 0 ${props => props.theme.spacing(1.5)}px 
```

Comment:
I think it would look nice, if we added some padding on the left to the input fields.

Reply:
done